opentelemetry/src/common.rs

```diff
@@ -462,6 +462,28 @@ impl InstrumentationScope {
             attributes: None,
         }
     }
+
+    /// Instrumentation library name.
+    pub fn name(&self) -> &str {
+        &self.name
+    }
```

Comment:
nit - make all these getters #[inline], just to be explicit. Even though compiler should be smart enough to do that.